src/client.ts

```diff
@@ -2303,6 +2303,24 @@ export class StreamChat<
     });
   }
 
+  /**
+   * unblockMessage - unblocks message blocked by automod
+   *
+   * Note: Do not use this.
+   * It is present for internal usage only.
+   * This function can, and will, break and/or be removed at any point in time.
+   *
```

Comment:
```suggestion
   *
   * @private
```

Reply:
Thanks for this, will add for other new private methods.

Reply:
Updated, please check.